docs/docs/7-migration-guide/migration-from-v8-to-v9.md

```diff
@@ -46,3 +46,104 @@ Example:
 ```
 
 </details>
+
+## Site
+
+### 🤖 Upgrade peer dependencies
+
+The following upgrade script will update peer dependency versions and make some minor changes in the code.
+
+:::note Execute the following upgrade script:
+
+```sh
+npx @comet/upgrade@latest v9/site/before-install
+```
+
+:::
+
+<details>
+
+<summary>Updates handled by this batch upgrade script</summary>
+
+#### ✅ Next.js
+
+Upgrade all your dependencies to support Next.js v15
+
+<details>
+
+<summary>Handled by @comet/upgrade</summary>
+
+:::note Handled by
+
+```sh
+npx @comet/upgrade@latest v9/site/before-install/update-next-dependencies.ts
+```
+
+:::
+
+```diff title=site/package.json
+{
+    "dependencies": {
+-       "@next/bundle-analyzer": "^14.2.30",
++       "@next/bundle-analyzer": "^15.5.4",
+-       "next": "^14.2.30",
+-       "react": "^18.3.1",
+-       "react-dom": "^18.3.1",
++       "next": "^15.5.4",
++       "react": "^19.2.0",
++       "react-dom": "^19.2.0",
+    },
+    "devDependencies": {
+-       "@types/react": "^18.3.23",
+-       "@types/react-dom": "^18.3.7",
++       "@types/react": "^19.2.0",
++       "@types/react-dom": "^19.2.0",
+    }
+}
+```
+
+</details>
+
+</details>
+
+### Add `next-env.d.ts` to `eslint.config.mjs`
+
+```diff title="site/eslint.config.mjs"
+/** @type {import('eslint')} */
+const config = [
+    {
+-       ignores: ["**/**/*.generated.ts", "dist/**", "lang/**", "lang-compiled/**", "lang-extracted/**"],
++       ignores: ["**/**/*.generated.ts", "dist/**", "lang/**", "lang-compiled/**", "lang-extracted/**", "next-env.d.ts"],
```

Comment:
I could create an upgrade script for this, but executing the script probably takes longer than adding the line to the config file.